### ISSUE TITLE
Cant run go get reisen. ERROR: could not determine kind of name for C.av_bsf_receive_packet and C.av_bsf_send_packet

### ISSUE THREAD

**Issue by onos9** [zergon321/reisen#9]
Getting errors when I go get reisen using `go get github.com/zergon321/reisen`

using  `brew install libav` did not work for me, so tried building ffmpeg from source using `brew install ffmpeg --build-from-source`  which installed all the library and header files, now i get all the following libraries  in ` /usr/local/include`  and  ` /usr/local/lib`

- **libavformat**
- **libavcodec**
- **libavutil**
- **libswresample**
- **libswscale**

but I’m getting the following  error  after running  `go get github.com/zergon321/reisen`
`../../go/pkg/mod/github.com/zergon321/reisen@v0.1.3/media.go:207:12: could not determine kind of name for C.av_bsf_receive_packet`

`../../go/pkg/mod/github.com/zergon321/reisen@v0.1.3/media.go:199:12: could not determine kind of name for C.av_bsf_send_packet`

I can confirm that all dependencies are install correctly.
please whats happing and can anything be done to resolve this, thank you

**Comment by zergon321** [comment]
Apparently it cannot find some function symbols. What version of `ffmpeg` do you have on your device?

**Comment by onos9** [comment]
Version 5.0

```bash
➜  ~ ffmpeg -version 
ffmpeg version 5.0 Copyright (c) 2000-2022 the FFmpeg developers
built with Apple clang version 13.0.0 (clang-1300.0.29.30)
configuration: --prefix=/usr/local/Cellar/ffmpeg/5.0 --enable-shared --enable-pthreads --enable-version3 --cc=clang --host-cflags= --host-ldflags= --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libbluray --enable-libdav1d --enable-libmp3lame --enable-libopus --enable-librav1e --enable-librist --enable-librubberband --enable-libsnappy --enable-libsrt --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libspeex --enable-libsoxr --enable-libzmq --enable-libzimg --disable-libjack --disable-indev=jack --enable-videotoolbox
libavutil      57. 17.100 / 57. 17.100
libavcodec     59. 18.100 / 59. 18.100
libavformat    59. 16.100 / 59. 16.100
libavdevice    59.  4.100 / 59.  4.100
libavfilter     8. 24.100 /  8. 24.100
libswscale      6.  4.100 /  6.  4.100
libswresample   4.  3.100 /  4.  3.100
libpostproc    56.  3.100 / 56.  3.100
```

**Comment by zergon321** [comment]
Fixed in [v0.1.4](https://github.com/zergon321/reisen/releases/tag/v0.1.4).

**Comment by onos9** [comment]
I can confirm that its perfectly working now great job thanks 👍
